src/main/java/seedu/address/model/RoomList.java

```diff
@@ -96,6 +95,7 @@ private void addRooms() {
                 internalList.remove(numOfRooms);
             }
         }
+        internalList.setAll(rooms);
```

Comment:
I would like to understand why this statement is necessary here.

Reply:
because initially it was just `internalList.add(sth)` and it wasnt updating the UI unless i change it to setAll

Reply:
Might be good to add a comment there to indicate that it refreshes UI.

Reply:
okay sure